### PR TITLE
Feature/simulate transaction

### PR DESCRIPTION
**Pull request by yousiefc**
# Description

This PR includes the following proposed change(s):

- Add timing changes to simulate transaction call

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?
```bash
yarn lint
yarn test -u
```

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+node_modules
+ai-reviewer-admin/node_modules
+efiling-demo/node_modules
+efiling-frontend/node_modules
+build
+.dockerignore
+Dockerfile
+Dockerfile.prod
+.git
```
Comment (KDNeufeld) [inline src/frontend/.dockerignore:9]: Based on my research, this .dockerignore file is in the wrong directory.  I think this file can be removed.

https://medium.com/@LihauTan/took-me-hours-to-realise-why-docker-build-ignores-my-dockerignore-and-this-is-what-ive-learned-2f87c770ea9c

From what I understand, the file must be located **at the root directory of the context**.  For us, this means placing a file like this in each of the UI subfolders.  For this project, only in src/frontend/ai-reviewer-admin

File: src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js [hunk 2]
```diff
@@ -125,13 +125,11 @@ export default function DocumentTypeEditor() {
           )}
           <Button
             label="Cancel"
-            data-testid="cancel-config-btn"
```
Comment (KDNeufeld) [inline src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js:128]: I think Karthik specifically added this line recently for his testing.  Are you sure you need to remove this?

https://github.com/bcgov/jag-file-submission/commit/9350ba7442c4d0728bc0ed820cb1605a280a1a78

File: src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js [hunk 3]
```diff
@@ -125,13 +125,11 @@ export default function DocumentTypeEditor() {
...
             styling="btn btn-secondary pull-left"
             onClick={cancelConfig}
           />
           <Button
             label="Submit"
-            data-testid="submit-config-btn"
```
Comment (KDNeufeld) [inline src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js:134]: same as above

File: src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js (hunk 2 above)
Comment (karthikmsqa) [reply src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js:128]: If it's OK, I'd like to keep the testid. Thanks!

Comment (yousiefc) [reply src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js:128]: I'm not even sure why it was removed so all good

**Review by KDNeufeld — APPROVED** [review]

**Pull request merged**